scopesim/effects/psfs.py

```diff
@@ -104,7 +109,10 @@ def apply_to(self, obj, **kwargs):
                             image[iplane,] - bkg_level[iplane,],
                             kernel[iplane,], mode=mode)
 
-                obj.hdu.data = new_image + bkg_level
+                # This line also copied all the data, better to do the
+                # modifications in place.
+                #obj.hdu.data = new_image + bkg_level
+                obj.hdu.data -= bkg_level
```

Comment:
This used to be an addition and is now a subtraction? Also, what happened to `new_image`?

Reply:
Ugh sorry for wasting your time, I didn't run the test suite :-(

Reply:
Happens to me as well sometimes 🙃